docs/pipeline.rst

```diff
@@ -45,7 +45,7 @@ The Setup Phase
 
 During this phase the test will be set up for the currently selected system partition and programming environment.
 The :attr:`current_partition` and :attr:`current_environ` test attributes will be set and the paths associated to this test case (stage, output and performance log directories) will be created.
-A `job descriptor <regression_test_api.html#reframe.core.pipeline.RegressionTest.job>`__ will also be created for the test case containing information about the job to be submitted later in the pipeline.
+A :attr:`job descriptor <reframe.core.pipeline.RegressionTest.job>` will also be created for the test case containing information about the job to be submitted later in the pipeline.
```

Comment:
The problem with this syntax here is that "job descriptor" will be formatted in verbatim, which is not what we want. That's why we used the explicit link here. I will revert this change.

Reply:
@vkarak I wonder if there's a way to get what I want while keeping the style as a hyperlink?

Reply:
I haven't found a way that's why we resorted to this solution, which is a bit verbose.

Reply:
I'm just thinking that it's hardcoding the HTML format.

Reply:
Okay sounds good, I'll do it soon.